scripts/src/run.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+
+run_usage() {
+  cat << EOF
+
+  Usage:
+    ./gear.sh run <FLAG>
+    ./gear.sh run <SUBCOMMAND> [CARGO FLAGS]
+
+  Flags:
+    -h, --help      show help message and exit
+
+  Subcommands:
+    help            show help message and exit
+
+    node            runs gear-node
+    purge-chain     purges gear node chain
+    purge-dev-chain purges gear dev node chain
+
+EOF
+}
+
+run_node() {
+  RUST_LOG=gwasm=debug,gear=debug cargo run -p gear-node "$@"
```

Comment:
```suggestion
  RUST_LOG="gwasm=debug,gear=debug" cargo run -p gear-node "$@"
```

Reply:
A forget change `gear` on `gear-core-backend`, yeah, but quotes aren't necessary here

Reply:
You can just add them and this string will be much more understandable

Reply:
Ok. As you wish